Tests/SonarScanner.MSBuild.Tasks.IntegrationTests/TargetsTests/ImportBeforeTargetsTests.cs

```diff
@@ -257,6 +235,47 @@ private string EnsureDummyIntegrationTargetsFileExists()
             return fullPath;
         }
 
+        private string CreateProjectFile(string testSpecificProjectXml)
```

Comment:
Each class now has  _CreateProjectFile_ method, and there is quite a lot of overlap between them. I haven't tried to extract the common parts as part of this change, partly because I don't want to spend more time on it now, and partly because that would start adding complexity back in.
What I'm trying to do with this refactoring is to have a fairly simple method/XML template in each case that makes it easy to see/change what is set.